tgui/packages/tgui/interfaces/IntegratedCircuit/DisplayComponent.jsx

```diff
@@ -43,20 +44,33 @@ export class DisplayComponent extends Component {
       <Box {...rest}>
         <div ref={this.ref}>
           <Box
-            backgroundColor={component.color || 'blue'}
             py={1}
             px={1}
-            className="ObjectComponent__Titlebar"
+            className={classes([
+              'ObjectComponent__Titlebar',
+              `ObjectComponent__Category__${component.category || 'Unassigned'}`,
```

Comment:
you're writing this enough that it might be nicer to just make it a const outside the jsx block